README.md

```diff
@@ -187,22 +187,28 @@ By default, the samples run in near realtime mode. To set offline mode, set stre
   - The depedency libraries are Curl, OpenSSL, and Log4Cplus.
  
 ### CMake Arguments
-You can pass the following options to `cmake ..`.
-
-* `-DBUILD_GSTREAMER_PLUGIN` -- Build kvssink GStreamer plugin
-* `-DBUILD_JNI` -- Build C++ wrapper for JNI to expose the functionality to Java/Android
-* `-DBUILD_DEPENDENCIES` -- Build depending libraries from source
-* `-DBUILD_TEST=TRUE` -- Build unit/integration tests, may be useful for confirm support for your device. `./tst/producerTest`
-* `-DCODE_COVERAGE` --  Enable coverage reporting
-* `-DCOMPILER_WARNINGS` -- Enable all compiler warnings
-* `-DADDRESS_SANITIZER` -- Build with AddressSanitizer
-* `-DMEMORY_SANITIZER` --  Build with MemorySanitizer
-* `-DTHREAD_SANITIZER` -- Build with ThreadSanitizer
-* `-DUNDEFINED_BEHAVIOR_SANITIZER` Build with UndefinedBehaviorSanitizer
-* `-DALIGNED_MEMORY_MODEL` Build for aligned memory model only devices. Default is OFF.
-* `-DBUILD_LOG4CPLUS_HOST` Specify host-name for log4cplus for cross-compilation. Default is OFF.
-* `-DCONSTRAINED_DEVICE` Set the thread stack size to 0.5MB, needed for Alpine builds
-
+You can pass the following CMake options:
```

Comment:
You should add "additional". We can pass in things supported by CMake itself, like CMAKE_INSTALL_PREFIX or CMAKE_EXE_LINKER_FLAGS

Reply:
Things specific to our project

Reply:
Makes sense, adding